tox.ini

```diff
@@ -1,12 +1,12 @@
 [tox]
 envlist =
-    {3.10,3.11,3.12}-test{,-alldeps,-shapely,-specutils}{,-cov}
-    {3.10,3.11,3.12}-test-numpy{123,124,125,126}
-    {3.10,3.11,3.12}-test-{numpy,astropy,linetools,ginga}dev
+    {3.11,3.12}-test{,-alldeps,-shapely,-specutils}{,-cov}
+    {3.11,3.12}-test-numpy{124,125,126,200}
```

Comment:
NumPy 2.1 has been released... do we want to test against it, too?

Reply:
Fair point.  Added!